### PR TITLE
Remain with cyanobacteria only in euphotic zone.

### LINKED ISSUE #395: Remove MLD switch for cyanobacteria - making euphotic zone N2 uptake remaining default
Hi @TomasTorsvik, 
I was briefly discussing with @JorgSchwinger about the next steps to be taken towards the iHAMOCC version for NorESM2.5 (as summarized in #340). We introduced those switches for cyanobacteria N2 uptake taking place in either the MLD or the euphotic zone only to enable better technical comparability between the isopycnic and the hybrid coordinates in BLOM. @TomasTorsvik : are they still needed or is this an outdated task and the switch can be removed in favor of cyanobacteria uptake only in the euphotic zone?

### PR DESCRIPTION
**Pull request by jmaerz**
closes #395

### CODE REVIEW

File: hamocc/mo_cyano.F90 [hunk 1]
```diff
@@ -75,7 +75,7 @@ subroutine cyano(kpie,kpje,kpke,kbnd,pddpo,omask,ptho)
     do j=1,kpje
       do i=1,kpie
         if (omask(i,j) > 0.5) then
-          do k=1,merge(kwrbioz(i,j),kmle(i,j),leuphotic_cya) ! if leuphotic_cya=.true., do bluefix only in euphotic zone
+          do k=1,kwrbioz(i,j) ! if leuphotic_cya=.true., do bluefix only in euphotic zone
```
Comment (TomasTorsvik) [inline hamocc/mo_cyano.F90]:
```suggestion
          do k=1,kwrbioz(i,j) ! do bluefix only in euphotic zone
```
The comment on this line can also be changed (or removed)

**Review by TomasTorsvik — APPROVED** [review]
@jmaerz - looks fine to me

**Review by JorgSchwinger — APPROVED** [review]
Thanks you, looks good to me!

File: hamocc/mo_cyano.F90 (hunk 1 above)
Comment (jmaerz) [reply hamocc/mo_cyano.F90]: Recognized it, when you approved :-) - sorry for that. Done.

**Pull request merged**